src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -226,12 +226,13 @@ export const selectPortfolioFiatBalanceByFilter = createCachedSelector(
     if (assetId && accountId) return portfolioAccountFiatbalances?.[accountId]?.[assetId] ?? '0'
     if (!assetId && accountId) {
       const accountBalances = portfolioAccountFiatbalances[accountId]
-      const totalAccountBalances = Object.values(accountBalances).reduce(
-        (totalBalance: string, fiatBalance: string) => {
-          return bnOrZero(totalBalance).plus(fiatBalance).toFixed(2)
-        },
-        '0',
-      )
+      const totalAccountBalances =
+        Object.values(accountBalances).reduce(
+          (totalBalance: string, fiatBalance: string | undefined) => {
```

Comment:
Remove callback typings, not true anymore

Reply:
sorry i'm confused

Reply:
`totalBalance` can now be `string | undefined` and will now be properly inferred

Reply:
@gomesalexandre fixed in [`3291bea` (#4251)](https://github.com/shapeshift/web/pull/4251/commits/3291beac8489b10f1d9ef9b49de105ed92fecfff)